src/app/collections/collection-sheet-status/collection-sheet-status.component.html

```
@@ -0,0 +1,251 @@
+        "
+      >
+        <mat-icon style="margin-right: 8px; color: #1565c0">group</mat-icon>
+        <span style="font-weight: bold; font-size: 16px; flex: 1">{{ group.groupName }}</span>
+        <span style="font-size: 14px; color: #1565c0; font-weight: bold; margin-right: 16px">
+          Total: {{ group.groupTotal | number: '1.2-2' }}
+        </span>
+        <span style="font-size: 13px; color: #666">{{ group.clients.length }} client(s)</span>
+      </div>
+
+      <!-- Clients within Group -->
+      <div *ngFor="let client of group.clients" style="margin-bottom: 8px; margin-left: 16px">
```

HTMLHint (1.9.1)

[warning] Inline style [  style="margin-bottom: 8px; margin-left: 16px" ] cannot be used.

(inline-style-disabled)